test/summarize.test.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs').promises;
+const summarize = require('../summarizer/summarize');
+
+
+describe('summarize tests...', () => {
+  let inputData;
+  let predicted;
+  let expected;
+
+  beforeAll(async () => {
+    inputData = await fs.readFile('./summarizer/input.txt', 'utf-8');
+    expected = await fs.readFile('./summarizer/expected.txt', 'utf-8');
+  });
+
+  test('summarize returns expected value', async () => {
+    predicted = await summarize(inputData, 5);
+    return expect(predicted).toBe(expected);
+  });
+
+  test('summarize returns string', async () => expect(typeof predicted).toBe('string'));
+
+  test('summarize returns correct number of lines', () => expect(predicted.split('.').length).toBe(6));
```

Comment:
What does `predicted` look like that you have to call split?  I don't follow the logic here.

Reply:
'Predicted' should be returning 5 sentences since I pass 5 sentences as a parameter for summarize. So I split 'predicted' in order to get the number of sentences. The module I use requires that a number of sentences be specified to produce a summary.

Reply:
Ah, I see.  Defining a sentence as "ends with a `.`" could be problematic, if a sentence includes something like "I'm working on a file named foo.js in my course."  Probably doing `/\. {1,2}/` for "period followed by 1 or 2 spaces" would be more accurate.  That still leaves out sentences that end with a `;`.  Anyway, this gets into the weed.  If you want to tighten up your `split()` to deal with the periods too, that's fine. Otherwise, we'll go with what you have here.